build/squire-raw.js

```diff
@@ -2254,17 +2254,33 @@ var onDrop = function ( event ) {
     while ( l-- ) {
         switch ( types[l] ) {
         case 'text/plain':
+        case 'Text': // IE Specific
             hasPlain = true;
             break;
         case 'text/html':
             hasHTML = true;
             break;
         default:
-            return;
+            break;
         }
```

Comment:
The `return` is intended to make it ignore the drop if it has content that's not text or HTML, on the presumption this isn't a text drag. What is the reasoning behind your change here? I don't think you want `break`, whatever the intention; maybe just delete the `default` case depending what you're going for.